server/src/main/java/com/objectcomputing/checkins/services/action_item/ActionItemRepository.java

```diff
@@ -17,6 +20,12 @@
 
     List<ActionItem> findByCreatedbyid(UUID uuid);
```

Comment:
is it safe to remove findByCheckinid, findByCreatedbyid?

Reply:
I was going to keep these because the search is going to get ordering in my next story, but on further thought I can't come up with a use case for keeping these non-ordered methods.

More importantly, removing these exposed that some of the unit tests needed further alteration. They were passing fine, they were just kidna funky.